uiTests/src/test/java/com/virtuslab/gitmachete/uitest/UITestSuite.java

```diff
@@ -59,20 +59,7 @@ public void openProjectAndAwaitIdle() {
   // Thus, we can't store any state between the invocations.
 
   @Test
-  public void toggleListingCommits() {
-    int branchRowsCount = callJs("project.refreshModelAndGetRowCount()");
-    // There should be exactly 6 rows in the graph table, since there are 6 branches in machete file,
-    // as set up via `super(SETUP_WITH_SINGLE_REMOTE)`.
-    Assert.assertEquals(6, branchRowsCount);
-
-    runJs("project.toggleListingCommits()");
-    int branchAndCommitRowsCount = callJs("project.refreshModelAndGetRowCount()");
-    // 6 branch rows + 7 commit rows
-    Assert.assertEquals(13, branchAndCommitRowsCount);
-  }
-
-  @Test
-  public void skipNonExistentBranches() {
+  public void skipNonExistentBranchesAndToggleListingCommits() {
```

Comment:
Merged the tests... to limit test running time at least a bit (since there's a significant overhead on creating/opening a new project in each test)